.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.7.4
+    hooks:
+      - id: pyupgrade
+        args: ["--py36-plus"]
+
+  - repo: https://github.com/psf/black
+    rev: 20.8b1
+    hooks:
+      - id: black
+        args: ["--target-version", "py36"]
+
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.6.4
+    hooks:
+      - id: isort
+
+  - repo: https://gitlab.com/pycqa/flake8
```

Comment:
```suggestion
  - repo: https://github.com/pycqa/isort
    rev: 5.6.4
    hooks:
      - id: isort

  - repo: https://gitlab.com/pycqa/flake8
```

```suggestion
  - repo: https://github.com/PyCQA/isort
    rev: 5.6.4
    hooks:
      - id: isort

  - repo: https://gitlab.com/PyCQA/flake8
```

Reply:
I used what the websites actually use:

* GitHub is `PyCQA`: `https://github.com/PyCQA/isort`

* GitLab is `pycqa`: `https://gitlab.com/pycqa/flake8`

Reply:
The suggestions are only about capitalizations, not gitlab vs github. Only the GitHub one seems correct, though.

Reply:
Oh woops